src/oneprovision/lib/provision.rb

```diff
@@ -186,35 +208,33 @@ def create(config, cleanup, timeout)
 
                 create_resources(cfg, cid)
 
-                if cfg['hosts'].nil?
-                    puts "ID: #{@id}"
-
-                    return 0
-                end
-
                 create_hosts(cfg, cid)
 
-                # ask user to be patient, mandatory for now
-                STDERR.puts 'WARNING: This operation can ' \
-                    'take tens of minutes. Please be patient.'
+                create_virtual_resources(cfg)
```

Comment:
Virtual resources should be created at the end after hosts are created and configured.

++ create steps order in the docs must be updated